README.md

```diff
@@ -2,16 +2,16 @@
 
 #### Project Build Status: &nbsp; [![Build Status](https://api.travis-ci.org/ReadyResponder/ReadyResponder.svg?branch=development)](https://travis-ci.org/ReadyResponder/ReadyResponder)
 
-### This application aids volunteer organizations to manage personnel, equipment and scheduling.
+### This application aids volunteer organizations to manage personnel, equipment, and scheduling.
 
-The project was inspired by [Sandi Metz's call for programmers to aid their communities](https://www.youtube.com/watch?feature=player_detailpage&v=fhpT6Pc4AqM#t=1931) .  This project in particular looks to lessons learned in response to emergencies that inspired the National ICS program.  It has often been found that there is plentiful equipment and personnel, but not the organization to know what was available nor the ability to manage it.
+The project was inspired by [Sandi Metz's call for programmers to aid their communities](https://www.youtube.com/watch?feature=player_detailpage&v=fhpT6Pc4AqM#t=1931) .  This project, in particular, looks to lessons learned in response to emergencies that inspired the National ICS program.  It has often been found that there are plentiful equipment and personnel, but not the organization to know what was available nor the ability to manage it.
 
 The goal of Ready Responder is to offer volunteer groups a program that allows them to track their resources and personnel, especially during emergencies or multi-day events.  This application might be used by volunteer firefighters, auxiliary police, Medical Reserve Corp (MRC), CERT organizations, amateur radio operators (ARES/RACES), church based relief groups, shelter managers or even science-fiction conventions.
 
 #### Current Features:
-* Web-based user interface, available from both desktop and mobile
-* Tracks complete data of personnel, including attendance, responsiveness and training
-* Tracks equipment, including serial numbers, sources, grants and service records
+* Web-based user interface, available from both desktop, and mobile
```

Comment:
HAHA. Don't worry these errors are pretty common, and after all, we aren't English majors, are we?